From 24f2930ea178aba0122ab542e197577b5391aeb5 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?= <avarab@gmail.com>
Date: Thu, 2 Sep 2010 17:42:05 +0000
Subject: [PATCH 09/25] gettextize: git-format-patch --cover-letter message
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Make the "SUBJECT HERE" and "BLURB HERE" output in git-format-patch
translatable. This is going to be edited by a human anyway.

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 builtin/log.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/builtin/log.c b/builtin/log.c
index 460353b..6a4a8f7 100644
--- a/builtin/log.c
+++ b/builtin/log.c
@@ -826,7 +826,7 @@ static void make_cover_letter(struct rev_info *rev, int use_stdout,
 {
 	const char *committer;
 	const char *subject_start = NULL;
-	const char *body = "*** SUBJECT HERE ***\n\n*** BLURB HERE ***\n";
+	const char *body = _("*** SUBJECT HERE ***\n\n*** BLURB HERE ***\n");
 	const char *msg;
 	const char *extra_headers = rev->extra_headers;
 	struct shortlog log;
-- 
1.7.2.2.614.g1dc9

